images/git/main.tf

```diff
@@ -0,0 +1,70 @@
+terraform {
+  required_providers {
+    apko = {
+      source                = "chainguard-dev/apko"
+      configuration_aliases = [apko.alpine]
+    }
+  }
+}
+
+locals {
+  accounts = toset(["nonroot", "root"])
+}
+
+variable "target_repository" {
+  description = "The docker repo into which the image and attestations should be published."
+}
+
+module "dev" { source = "../../tflib/dev-subvariant" }
+
+module "tagger" {
+  source = "../../tflib/tagger"
+
+  depends_on = [
+    module.test-latest-alpine,
+    module.test-latest-alpine-dev,
+    module.test-latest-wolfi,
+    module.test-latest-wolfi-dev,
+  ]
+
+  tags = merge(
+    # Tagging format for alpine/musl variant:
+    # - latest
+    # - latest-dev
+    # - latest-root
+    # - latest-root-dev
+    # - $v
+    # - $v-dev
+    # - root-$v
+    # - root-$v-dev
+    # Nonroot alpine variant
+    { for t in toset(concat(["latest"], module.version-tags-alpine["nonroot"].tag_list)) : t => module.latest-alpine["nonroot"].image_ref },
+    { for t in toset(concat(["latest"], module.version-tags-alpine["nonroot"].tag_list)) : "${t}-dev" => module.latest-alpine-dev["nonroot"].image_ref },
```

Comment:
I think my only micro-nit would be that, given that we use `["..."]` as a subscript, and this is the only version that does the `concat`, I wonder if it would be slightly clearer to do `latest` explicitly on its own rows.

That said, I agree this is about as clear as we're going to get 🙏

Reply:
no nit is too small!